pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -30,6 +27,23 @@
 
 
 def solve_OA_master(solve_data, config):
```

Comment:
@michaelbynum here is the function where we solve the Master Problem for (global) OA. I will point you to the piece of code that adds the affine cuts from MC++.